### PR TITLE
Implement multibands for operators

### PR DESCRIPTION
**Pull request by michaelmattig**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by michaelmattig** [comment]
[[Raster-Stacking-Placeholder]](https://trello.com/c/jHuUp2Oc/1566-raster-stacking-placeholder)

File: operators/src/processing/temporal_raster_aggregation/temporal_aggregation_operator.rs [hunk 1]
```diff
@@ -257,14 +257,18 @@ where
     #[allow(clippy::too_many_lines)]
     fn create_subquery_adapter_stream_for_single_band<'a>(
         &'a self,
-        query_rect_to_answer: RasterQueryRectangle,
-        band_idx: u32,
+        query: RasterQueryRectangle,
```
Comment (jdroenner) [inline operators/src/processing/temporal_raster_aggregation/temporal_aggregation_operator.rs:260]: cant we use that partial query here?

File: operators/src/processing/meteosat/temperature.rs [hunk 2]
```diff
@@ -302,9 +301,13 @@ where
         query: RasterQueryRectangle,
         ctx: &'a dyn QueryContext,
     ) -> Result<BoxStream<'a, Result<Self::Output>>> {
-        let src = self.source.query(query, ctx).await?;
-        let rs = src.and_then(move |tile| self.process_tile_async(tile, ctx.thread_pool().clone()));
-        Ok(rs.boxed())
+        stack_individual_aligned_raster_bands(&query, ctx, |query, ctx| async move {
```
Comment (jdroenner) [inline operators/src/processing/meteosat/temperature.rs:304]: why is this needed for this operator but not for the other MSG Operators?

Comment (michaelmattig) [reply operators/src/processing/meteosat/temperature.rs]: I initially thought it was needed for all msg operators, but as they just map tiles and don't care about the band, I removed it again. I only forgot it here: 02d77a9e4c6b54b8a0ee8a272c89873799792d19

File: operators/src/processing/temporal_raster_aggregation/temporal_aggregation_operator.rs (hunk 1 above)
Comment (michaelmattig) [reply operators/src/processing/temporal_raster_aggregation/temporal_aggregation_operator.rs:260]: do you mean `PartialQueryRectangle`? If so, then no, because it doesn't include the band but the subquery still needs to know it.

**Review by jdroenner — APPROVED** [review]
